cli/command/stack/services.go

```diff
@@ -1,12 +1,17 @@
 package stack
 
 import (
+	"fmt"
+
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/command/service"
+	"github.com/docker/cli/cli/command/stack/formatter"
 	"github.com/docker/cli/cli/command/stack/kubernetes"
 	"github.com/docker/cli/cli/command/stack/options"
-	"github.com/docker/cli/cli/command/stack/swarm"
+	swarmStack "github.com/docker/cli/cli/command/stack/swarm"
 	cliopts "github.com/docker/cli/opts"
+	"github.com/docker/docker/api/types/swarm"
```

Comment:
(see other comment above) and change this one to

```suggestion
	swarmtypes "github.com/docker/docker/api/types/swarm"
```